docs/PR_GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request Guide
+
+## General development rules:
+
+- All the changes of the code base are implemented through pull requests (PR) on master or other branches
+- When all the features for an upcoming release are implemented and are stable enough, a new release branch is made from master. After that, all app versions are updated in the master branch in files: ./omim/xcode/common.xcconfig and ./omim/android/gradle.properties
+- Only PR with bugfix should be done on release branches. If it’s necessary it’s permitted to cherry-pick such bugfix PRs to master
+- When a release is published one of developers makes a PR with all new commits of the release to master
+- All the further commits to release branch should be cherry-picked to the master by commits author
+- When a PR to any branch is about to merge, rebase and merge option should be used to make the code source tree flatter
+
+
+## Writing good PR:
+
+- PR should be small and reflect only one idea, a feature, a bugfix or a refactoring. In most cases a PR of 100 lines or less is considered as a PR of normal size and a PR of 1000 lines as big one
```

Comment:
let's add serial comma here (and in other places): ...a bugfix, or a refactoring

Reply:
Done